shoop/simple_pricing/migrations/0002_remove_simpleproductprice_includes_tax.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('simple_pricing', '0001_initial'),
```

Comment:
I.e. no need for changes to this IMHO. I don't think adding a data migration has much value.